test/ModuleInterface/inlinable-function.swift

```diff
@@ -0,0 +1,242 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -emit-module -o %t/Test.swiftmodule -emit-interface-path %t/Test.swiftinterface -module-name Test %s
+// RUN: %FileCheck %s < %t/Test.swiftinterface
+// RUN: %target-swift-frontend -emit-module -o /dev/null -merge-modules %t/Test.swiftmodule -disable-objc-attr-requires-foundation-module -emit-interface-path - -module-name Test | %FileCheck %s
+
+// CHECK: public struct Foo : Hashable {
+public struct Foo: Hashable {
+  // CHECK: public var inlinableGetPublicSet: [[INT:(Swift.)?Int]] {
+  public var inlinableGetPublicSet: Int {
+  // CHECK: @inlinable get {
+  // CHECK-NEXT: return 3
+  // CHECK-NEXT: }
+    @inlinable
+    get {
+      return 3
+    }
+    // CHECK-NEXT: set[[NEWVALUE:(\(newValue\))?]]{{$}}
+    set {
+      print("I am set to \(newValue)")
+    }
+  }
+
+  // CHECK: public var noAccessors: [[INT]]{{$}}
+  public var noAccessors: Int
+
+  // CHECK: public var hasDidSet: [[INT]] {
+  public var hasDidSet: Int {
+    // CHECK-NEXT: @_transparent get{{$}}
+    // CHECK-NEXT: set[[NEWVALUE]]{{$}}
+    // CHECK-NOT: didSet
+    didSet {
+      print("b set to \(hasDidSet)")
+    }
+    // CHECK-NEXT: }
+  }
+
+
+  // CHECK: @_transparent public var transparent: [[INT]] {
+  // CHECK:   return 34
```

Comment:
We're planning to change this to always use `get {`, right?

Reply:
Yep, should I go ahead and roll that into this patch?

Reply:
A follow-up is fine; just wanted to have it on record.